packages/checkouts/config/environment.js

```diff
@@ -1,5 +1,18 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  var ENV = {};
+
+    ENV['spree'] = {
+      apiHost: 'http://testing.spree-ember.com',
+      namespace: "api/ams",
+      mount: "/",
+      productsPath: "products",
+      cartPath: "cart",
+      checkoutPath: "checkout",
+      ordersPath: "orders",
+      taxonsPath: "t"
+    };
```

Comment:
These are technically configs that live in the storefronts (and core for the api/namespace settings) package - they don't belong here.